mne/tests/common.py

```diff
@@ -46,3 +47,17 @@ def assert_meg_snr(actual, desired, min_tol, med_tol=500., msg=None):
     snr = np.median(snrs)
     assert_true(snr >= med_tol, 'SNR median %0.2f < %0.2f%s'
                 % (snr, med_tol, msg))
+
+
+def assert_dig_allclose(info_py, info_bin):
+    # test dig positions
+    dig_py = info_py['dig']
+    dig_bin = info_bin['dig']
+    assert_equal(len(dig_py), len(dig_bin))
```

Comment:
we didn't assert the length because the files have different hsp lengths but they can be disentangled in the `info['dig']` by selecting all the ones not dig points.